.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "prettier"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "plugins": [
+    "flowtype",
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": "error"
```

Comment:
I'm not super knowledgable about Eslint. I just followed the suggested config from the Prettier site.